lib/hooks/cast.cpp

```diff
@@ -93,7 +93,8 @@ class CastHook : public MultiSignalHook {
 static char n[] = "cast";
 static char d[] = "Cast signals types";
 static HookPlugin<CastHook, n, d,
-                  (int)Hook::Flags::NODE_READ | (int)Hook::Flags::PATH>
+                  (int)Hook::Flags::NODE_READ | (int)Hook::Flags::NODE_WRITE |
+                      (int)Hook::Flags::PATH>
```

Comment:
Is this indentation really correct?

Reply:
This is what clang-format enforces. I don't really get why. I'll separate the flags to make it look nicer.